README.md

```diff
@@ -286,6 +274,33 @@ console.error('error log message!')
 ```
 For more information check the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/console) on `console` object.
 
+### Getting Rugged By The WiFi?! 
+Run a local testnet with [`docker` support](https://docs.docker.com/desktop/). All you need to do to get a local testnet started is run:
+
+```shell
+npm run dev
+```
```

Comment:
It may got an error if the user is already using port `8000`. We should mentioned that in the doc.

Reply:
Done